<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>150</width>
				<height>189</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Unaverage</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="label_6">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Items</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_5">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="Label_ItemsCount">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>0</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_02" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_4">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPlainTextEdit" name="Text_Items">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_03" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_2">
							<property name="spacing">
								<number>4</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="Label_Amount">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Amount</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_7">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QSpinBox" name="Int_Amount">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="minimum">
										<number>1</number>
									</property>
									<property name="maximum">
										<number>50</number>
									</property>
									<property name="value">
										<number>16</number>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_04" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_3">
							<property name="spacing">
								<number>4</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="Label_Blend">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Blend</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_8">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QDoubleSpinBox" name="Float_Blend">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>6</number>
									</property>
									<property name="maximum">
										<double>9999.000000000000000</double>
									</property>
									<property name="singleStep">
										<double>0.050000000000000</double>
									</property>
									<property name="value">
										<double>0.500000000000000</double>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_05" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_5">
							<property name="spacing">
								<number>4</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="Label_Iterations">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Iterations</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_9">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QSpinBox" name="Int_Iterations">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="minimum">
										<number>1</number>
									</property>
									<property name="maximum">
										<number>99999</number>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_6">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_Apply">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Apply</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>